comics/comics/darklegacy.py

```diff
@@ -11,7 +11,7 @@ class ComicData(ComicDataBase):
 
 
 class Crawler(CrawlerBase):
-    history_capable_date = "2006-12-09"
+    history_capable_days = 29 * 7  # 7 weekly releases
```

Comment:
This doesn't look right?

```suggestion
    history_capable_days = 29 * 7  # 7 monthly releases
```

Reply:
I put the wrong number in the comment, I corrected it in 58caeb5953df611255f0b14b59694f60ba66345c.
The feed currently has 33 releases, with one coming each Sunday. I don't know the limit for the feed, the page has more releases than the feed, so I don't think it would be correct to set a history_capable_date.